online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZSpinner.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 23/8/2022.
+ */
+
+package com.adyen.checkout.onlinebankingcz
+
+import android.content.Context
+import android.net.Uri
+import android.util.AttributeSet
+import android.view.View
+import androidx.browser.customtabs.CustomTabsIntent
+import com.adyen.checkout.components.model.payments.request.OnlineBankingCZPaymentMethod
+import com.adyen.checkout.components.ui.util.ThemeUtil
+import com.adyen.checkout.issuerlist.IssuerListSpinnerView
+
+class OnlineBankingCZSpinner @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : IssuerListSpinnerView<OnlineBankingCZPaymentMethod, OnlineBankingCZComponent>(context, attrs, defStyleAttr) {
+
+    override fun onComponentAttached() {
+        super.onComponentAttached()
+        termsAndConditionsTextView?.visibility = View.VISIBLE
```

Comment:
`IssuerListSpinnerView`  now has these terms and conditions views, but they do not fully function there. I would either make it fully working for `IssuerListSpinnerView` or have a special layout for `OnlineBankingCZSpinner`, so only this spinner does have terms and conditions. The last option has my preference as it makes sure existing spinners are not affected

Reply:
I tried to do another view for `OnlineBankingCZSpinner ` but I have to create a whole new component that implements a `ViewableComponent` and there's no use of `IssuerListSpinnerView` anywhere I mean every other Issuer component is using recyclerView instead.

Reply:
From what I understood, the other implementations of `IssuerListSpinnerView` can be used by merchants if they implement standalone components and don't want to use the `RecyclerView`s. It might be more work, but I still think it's a good idea to separate the terms and conditions stuff from the other spinners

Reply:
It's a lot of work I'll be redoing the whole thing 😅 because `IssuerListComponent` is not extending `ViewableComponent` so I'll redo the whole component and delegate